globus_cli/commands/endpoint/is_activated.py

```diff
@@ -29,14 +29,25 @@ def endpoint_is_activated(endpoint_id, until, absolute_time):
     def fail(deadline=None):
         exp_string = ''
         if deadline is not None:
-            exp_string = ' or will expire before {}'.format(deadline)
+            exp_string = ' or will expire before {} seconds'.format(deadline)
 
-        formatted_print(
-            res, simple_text=(
-                '{0} is not activated{1}\n'
-                'To activate, please go to the following page:\n\n'
-                '  https://www.globus.org/app/endpoints/{0}/activate\n'
-                ).format(endpoint_id, exp_string))
+        myproxy = supports_myproxy(res)
+
+        lines = [
+            "The endpoint is not activated{}.".format(exp_string),
+
+            "The endpoint supports Web{} activation:".format(
+                " and MyProxy" if myproxy else ""),
+
+            ("Use 'globus endpoint activate {} --web' "
+                "for Web activation.".format(endpoint_id)),
+
+            "{}".format(("Use 'globus endpoint activate {} --myproxy' "
+                         "for Myproxy activation."
+                         .format(endpoint_id)) if myproxy else "")
```

Comment:
You can just use the parenthetical expression, no need to `"{}".format(x)`, since that's the same as `x`.
I can totally see how with all of the tricks and games we play with `format()` this kind of thing sneaks up on you.